package.json

```
@@ -49,6 +49,9 @@
 		"hint",
 		"simple"
 	],
+	"dependencies": {
+		"confusing-browser-globals": "^1.0.9"
```

```suggestion
		"confusing-browser-globals": "1.0.9"
```

Since new addition to that module is technically a breaking change when linting as it will cause new lint violations.

---

Updated